src/routes/relay/entities/schemas/__tests__/relay.dto.schem.spec.ts

```diff
@@ -0,0 +1,133 @@
+import { RelayDtoSchema } from '@/routes/relay/entities/schemas/relay.dto.schema';
```

Comment:
Nit: there is a typo in the filename (`schem.spec.ts`) 😄

Reply:
🦅 Fixed in 65586e04007c76e036ebaa09a03c5e124d417e82.